.github/workflows/docker_release.yml

```diff
@@ -10,6 +10,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Filter Helm Releases
```

Comment:
Will want @nguyer's input on this. This assumes we might make separate GH releases for the chart itself, and so we needed a way to prevent the Docker image from being re-published if that was done.